src/table/resizer/styles.scss

```diff
@@ -50,6 +50,12 @@ th:not(:last-child) > .divider {
   inset-inline-end: calc(-1 * #{$handle-width} / 2);
   inline-size: $handle-width;
   z-index: 10;
+
+  // Allow touch devices to use the resizer horizontally.
+  // The lint isn't correct: "pan-y" and "pinch-zoom" values are supported.
```

Comment:
Is it possible to upgrade the related dependency, instead of writing a comment?

Seeing [how often we ignore that rule already](https://github.com/search?q=repo%3Acloudscape-design%2Fcomponents%20no-unsupported-browser-features&type=code) it is also possible to remove that rule all together